pkg/deployment/config.go

```diff
@@ -64,4 +87,8 @@ type Config struct {
 
 	// Repositories for which tag to digest resolving should be skipped
 	RegistriesSkippingTagResolving sets.String
+
+	// ProgressDeadlineSeconds is the time in seconds we wait for the deployment to
+	// be ready before considering it failed.
+	ProgressDeadlineSeconds time.Duration
```

Comment:
This shouldn't contain `Seconds` if it's a `Duration`.

Reply:
I'd align with what we do for the request timeout knob (should be one or the other)

Reply:
Done.